docs/en/operations/requirements.md

```diff
@@ -0,0 +1,52 @@
+# Requirements
+
+## CPU
+
+In case of installation from prebuilt deb-packages use CPU with `x86/64` architecture and `SSE 4.2` instructions support. If you build ClickHouse from sources, you can use other processors.  
+
+ClickHouse implements parallel data processing and uses all the hardware resources available. When choosing a processor, take into account that ClickHouse works more efficient at configurations with a large number of cores but lower clock rate than at configurations with fewer cores and a higher clock rate. For example, 16 cores with 2600 MHz is preferable than 8 cores with 3600 MHz.
+
+Use of **Turbo Boost** and **hyper-threading** technologies is recommended. It significantly improves performance with a typical load.
+
+## RAM
+
+Use 4GB of RAM as a minimum. The ClickHouse server can run with a much smaller amount of RAM, but it requires memory for queries processing.
```

Comment:
It's not a hard minimum. We'd better say that's it's a recommended minimum for running non-trivial queries.